### PR TITLE
Created AppendTextAssetCommand

### PR DESCRIPTION
**Pull request by openbots-dh**
- Adds AppendTextAssetCommand to the Asset Commands and adds AppendAsset() method to AssetMethods.cs

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.Asset/AppendTextAssetCommand.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	[Serializable]
+	[Category("Asset Commands")]
+	[Description("This command appends text to the end of a text Asset in OpenBots Server.")]
+	public class AppendTextAssetCommand : ScriptCommand
+	{
+		[Required]
+		[DisplayName("Text Asset Name")]
+		[Description("Enter the name of the Asset.")]
+		[SampleUsage("Name || {vAssetName}")]
+		[Remarks("This command will throw an exception if an asset of the wrong type is used")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Asset/AppendTextAssetCommand.cs:27]:
```suggestion
		[Remarks("This command will throw an exception if an asset of the wrong type is used.")]
```

**Review by openbots-ff — CHANGES_REQUESTED** [review]
Please add a test for this command, but otherwise, it looks great!

**Review by openbots-su — CHANGES_REQUESTED** [review]
As mentioned in the other comment, you just need to implement a test for the command. Otherwise, I don't see any issues in the command itself.

File: OpenBots.Commands/OpenBots.Commands.Asset/AppendTextAssetCommand.cs (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands/OpenBots.Commands.Asset/AppendTextAssetCommand.cs]: Period for remark added in b53b3e67306b416db58cefede369ad61d22da5b0

Tests for this command added in b6b2f68 and 6d31657

**Review by openbots-su — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Review by openbots-su — APPROVED** [review]

**Pull request merged**